### DISCUSSION TITLE
zh/guide/drivers/aliyundrive

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#31] [Announcements]
# zh/guide/drivers/aliyundrive

AList v3的文档

https://alist.nn.ci/zh/guide/drivers/aliyundrive.html

**Comment by dvdrm** [comment]
Failed create storage in database: UNIQUE constraint failed: x_storages.mount_path

**Reply by zu0322** [reply]
一样

**Reply by 1idjdn** [reply]
一样怎么解决

**Reply by aixiaofei** [reply]
挂载路径重复了，更换一下

**Comment by tiger4777** [comment]
Failed init storage: Post "https://auth.aliyundrive.com/v2/account/token": x509: certificate signed by unknown authority

**Reply by 695975525** [reply]
怎么解决

**Comment by Sky-seeker** [comment]
- **阿里云盘** 根文件夹 `ID` 挂载出错。
- 子文件夹ID如 `5fe01e1830601baf774e4827a9fb8fb2b5bf7940` ，根文件夹ID为 `root` ，但是挂载出错。
- 验证如下
  ![根文件夹](https://i.postimg.cc/rphpjzTQ/Image-2022-09-28-14-31-26.jpg)
  ![子文件夹](https://i.postimg.cc/Px95GcPD/Image-2022-09-28-14-32-17.jpg)
  ![Alist 阿里云盘](https://i.postimg.cc/NM5jzYfY/Image-2022-09-28-14-33-11.jpg)

**Reply by setvip** [reply]
二维码出不来

**Comment by hackerpluses** [comment]
令牌我用了12小时不到就过期了。。。。。用手机阿里网盘APP扫码得到的。
Post "https://auth.aliyundrive.com/v2/account/token": context deadline exceeded (Client.Timeout exceeded while awaiting headers)

**Reply by loomee** [reply]
个人是用这个说明提取的，目前还没遇到问题。
[阿里云盘列表AList部署教程-Cooluc-Blog](https://www.cooluc.com/archives/1392.html)

**Comment by elicip** [comment]
怎么才能刷新目录缓存，阿里云盘里都调整过目录了可是alist里还是显示老的目录结构

**Reply by yuzaii** [reply]
确实我也遇到了这个问题

**Reply by Kevin-1117** [reply]
alist网页端右下角有个刷新按钮，点一下就出来最新的文件内容了。（只在电脑端测试过。其他的自行测试。）

**Reply by ocean521** [reply]
手机端也可以，但貌似最好多点几次，个别时候没有全部刷新（比如多个文件夹的情况）

**Reply by cheduzi0060** [reply]
右下角那个按钮真是找了好几次才看到，哈哈，刷新确认解决！

**Comment by yiwang-base** [comment]
封号了   兄弟们！！！

**Reply by wbsnpc** [reply]
你的阿里云盘账号被封了吗

**Reply by Asuna125** [reply]
显示什么原因被封禁？

**Comment by vpei** [comment]
阿里云盘挂载后，目录能正常访问，打开视频，提示：failed get link: ForbiddenNoPermission.File:No Permission to acc
不知道这个是咋回事。

**Reply by vpei** [reply]
alist-3.2.0

**Reply by vpei** [reply]
是阿里云盘分享

**Reply by cbnui** [reply]
目前版本v3.3.0，也有和你出现一样的问题，但挂载好后，需要重启alist服务，即可访问，应该是个BUG

**Comment by qyidc** [comment]
目前我的qnap遇到一个问题，使用alist挂载阿里云一切正常，使用HybridMount挂载也正常，测试速度正常，测试速度后在云盘中可以看到HybridMount创建的测试文件，说明HybridMount工作也是正常的，在file station中也能够看到挂载的云盘，但是就是打不开。

**Reply by ocean521** [reply]
我也是通过这种方式，可以打开，我的问题是想用PLEX来浏览影视资源，PLEX找不到这个文件夹，不知道如何挂载到本地。

**Reply by zyuzone** [reply]
我是用https://github.com/iranee/qnap-alist-webdav/这个alist的qnap套件挂载阿里云盘，然后HybridMount挂载为本地文件夹。目前一切正常（包括file station），emby也能正常使用，速度还挺快，进度条拖动很流畅。

**Comment by nice1try** [comment]
3.2.0与3.1.0中挂载成功后可以查看目录 但是文件无法打开

**Reply by nice1try** [reply]
2小时后重新获取了token 在3.1.0版本成功播放

**Comment by yukun1** [comment]
状态: 
Post "https://auth.aliyundrive.com/v2/account/token": context deadline exceeded (Client.Timeout exceeded while awaiting headers)

**Comment by 2247481637** [comment]
上传只显示一个文件咋搞

**Comment by kooyeed** [comment]
可以删除，可以上传，就是不能下载。。。

**Comment by ykc1043** [comment]
获取不到文件
failed get objs: failed to list objs: Post "https://api.aliyundrive.com/v2/file/list": context deadline exceeded (Client.Timeout exceeded while awaiting headers)

**Reply by ykc1043** [reply]
之后无法添加存储
Post "https://api.aliyundrive.com/v2/user/get": context deadline exceeded (Client.Timeout exceeded while awaiting headers)

**Reply by ykc1043** [reply]
忘了说是阿里云盘

**Comment by Ceonta** [comment]
页面Potplayer按钮不能正常打开文件，显示文件不存在

**Reply by vpei** [reply]
> 页面Potplayer按钮不能正常打开文件，显示文件不存在

这个需要本地浏览器支持，同时本地还需要安装好相对应的播放器。

**Reply by Ceonta** [reply]
浏览器跟播放器都没问题，应该是新版的BUG，以前旧版点开就能播放的。

**Reply by 635621128** [reply]
WebDAV 策略改一下，不要用302重定向

**Reply by cxwsssj** [reply]
我也遇到了相同的问题，求解

**Comment by QINSSTRIVE** [comment]
failed get objs: failed to list objs: The input parameter share_id, drive_id is not valid. get file request drive_id and share_id are mutually exclusive, and must be passed only one

**Reply by QINSSTRIVE** [reply]
刷新令牌刷新引起，不知如何解决

**Comment by 635621128** [comment]
openwrt使用alist挂载阿里云盘和使用阿里云盘 WebDAV挂载速度差距很大，使用alist挂载的只有阿里云webdav的一半都不到

**Comment by xj9696** [comment]
阿里云盘挂载后如果文件夹名称中包含“/”，则不能打开

**Comment by iluofeng** [comment]
阿里云盘的pdf文件，苹果手机无法下载，点击下载直接在线打开了，没有弹出下载框

**Comment by JamesYang0304** [comment]
2022-11-25T05:17:19.709228Z  WARN aliyundrive_webdav::drive: refresh token failed, will wait and retry error=error sending request for url (https://websv.aliyundrive.com/token/refresh): error trying to connect: dns error: failed to lookup address information: Name does not resolve，一直报这个错误，要怎么解决？

**Comment by jackoma1989** [comment]
昨天，突然alist挂载的阿里云，在plex上播放失败，同样挂载的百度云没有问题。是因为官方对视频限速了？直接用阿里云tv版播放也没问题。更新了手机端的token也不行

**Comment by zxq1002** [comment]
HBS3同步了一段时间后失败了，访问http://192.168.x.x:5244/aliyundrive提示如下错误，无法列出文件，怎么解决。。
failed get objs: failed to list objs: Post "https://api.aliyundrive.com/v2/file/list": dial tcp: lookup api.aliyundrive.com: Try again

**Comment by iyexiaolou** [comment]
Failed init storage but storage is already created: Post "https://api.aliyundrive.com/v2/user/get": dial tcp *********:443: connect: connection refused

一直提示这个是什么意思？

**Comment by apang1945** [comment]
想问一下这个
failed get objs: failed to list objs: Post "https://api.aliyundrive.com/v2/file/list": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
是怎么了，我是国外的服务器，登录阿里云盘，然后刷新不出来

**Reply by cancundeyingzi** [reply]
响应超时

**Comment by peter0511** [comment]
Failed init storage: failed to refresh token: The input parameter refresh_token is not valid.

**Comment by cancundeyingzi** [comment]
昨天封号,提醒各位别玩过火.小玩一下不是问题,别搞太猛

**Reply by lastwidn** [reply]
干啥了被封了老铁，分享一下经验吧。

**Reply by cancundeyingzi** [reply]
怀疑是下载量

**Comment by cditmoo** [comment]
新的alist 3.72 windows版本里，没有阿里云盘的选项啦，用alist v2添加出错，failed get objs: failed to list objs: Post "/api/public/path": unsupported protocol scheme ""，应该是链接没填对

**Comment by wenge110110** [comment]
请教大佬：挂载的阿里云盘，视频只能电脑看，安卓和苹果手机上怎么都放不出来，我使用原站点IP+端口不行 反代的域名也不行 目前版本 3.8.0

**Reply by cancundeyingzi** [reply]
苹果手机看不了，正常。安卓我这边倒是能看。不过我用的是2的最后一个版本。三，我感觉是开到车。除了功能，其他美观性，便捷性全倒退。

**Comment by bluesky4485** [comment]
现在获取token有跨域的问题，二维码出不来

**Comment by Nanako718** [comment]
获取Token可以登陆网页版打开F12控制台输入JSON.parse(window.localStorage["token"]).refresh_token;

**Comment by STEVENTAN100** [comment]
请问各位群友，阿里云盘的秒传按钮有什么用处吗？

**Reply by evlic** [reply]
同问 + 1，准备上源码研究下，晚点等我消息

**Reply by evlic** [reply]
根据源码猜测是计算 md5/hash 一类的文件唯一标识，如果阿里云上有这个文件，直接调转存 API。


> drivers/aliyundrive/driver.go:172

```go

if d.RapidUpload {
		buf := bytes.NewBuffer(make([]byte, 0, 1024))
		io.CopyN(buf, file, 1024)
		reqBody["pre_hash"] = utils.GetSHA1Encode(buf.String())
		// 把头部拼接回去
		file.ReadCloser = struct {
			io.Reader
			io.Closer
		}{
			Reader: io.MultiReader(buf, file),
			Closer: file,
		}
}
```

**Comment by motian16** [comment]
这个是只能填自己的分享链接吧，刷新令牌必须是分享账号的么，我填其他账号的令牌提示
Failed init storage: The resource share_link cannot be found. Please check.

**Comment by wangminli** [comment]
点击获取token的时候一直显示“等待……”，无法展示二维码，是什么原因呢？

**Comment by ttxs8** [comment]
阿里云盘和阿里云盘分享，这两个有什么区别吗？

**Comment by andy841116** [comment]
多个阿里云盘账号怎么添加？几个不同的账号token添加后打开的内容是一样的。

**Comment by lrs95** [comment]
可不可以支持创建一个分享目录填写多个分享id

**Comment by 695975525** [comment]
failed get objs: failed to list objs: Post "https://api.aliyundrive.com/v2/file/list": x509: certificate signed by unknown authority  提示证书问题

**Comment by Jerry12228** [comment]
离线下载文件功能中，下载完成后从本地上传到对应存储时，可以限制上传速度吗？

**Comment by 695975525** [comment]
不可以吧



---Original---
From: ***@***.***&gt;
Date: Sat, Feb 11, 2023 13:19 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [alist-org/docs] zh/guide/drivers/aliyundrive (Discussion #31)




 
离线下载文件功能中，下载完成后从本地上传到对应存储时，可以限制上传速度吗？

**Comment by onelxzy** [comment]
今天出现的。报错failed get link: invalid X-Device-Id。更新token也无法使用

**Reply by zzzrrr-code** [reply]
一样

**Reply by sz173510443** [reply]
都这样了

**Reply by miaoz** [reply]
看來不是個例。

**Reply by YokioeXie** [reply]
是不是阿里 WebDev 关了？

**Reply by qqxpee** [reply]
刚想更新token，那看来不是token的问题了

**Reply by gaoxiao228** [reply]
同样

**Reply by guangguailulu** [reply]
同样，我也以为是token问题呢，更新了token也不行

**Reply by appe233** [reply]
昨天晚上弄好的阿里云盘，也发现是这样，看来不是我一个了

**Reply by xiyuyeyun** [reply]
目录可以访问，一旦播放资源就出现：failed get link: invalid X-Device-Id,同样，有知道如何解决的大神嘛？

**Reply by 1219ls** [reply]
一样

**Reply by xiyuyeyun** [reply]
现在恢复了吗？有没有人知道？回复一下呗

**Reply by cancundeyingzi** [reply]
> 现在恢复了吗？有没有人知道？回复一下呗

........访问[Releases]2分钟,问人5小时....

**Reply by xiyuyeyun** [reply]
解决了吗？

**Comment by sz173510443** [comment]
不清楚&nbsp; 之前一直正常 也没去动过阿里云盘&nbsp; 然后突然所有阿里盘的都失效了




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年2月13日(星期一) 中午11:53
收件人: ***@***.***&gt;; 
抄送: "Andy ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [alist-org/docs] zh/guide/drivers/aliyundrive (Discussion #31)





 
是不是阿里 WebDev 关了？

**Comment by qqxpee** [comment]
failed get link: invalid X-Device-Id，
可以访问文件夹，但是打开文件报这个错，
百度跟google都没有类似的话题，有大佬知道怎么回事嘛

**Reply by 695975525** [reply]
都一样，阿里也开始搞事了

**Comment by miaoz** [comment]
Github 今天有相关讨论，应该是阿里云盘改了设备认证方式，只能等 AList 更新了吧。
________________________________
发件人: qiuxueqi1994 ***@***.***>
发送时间: Monday, February 13, 2023 11:57:28 AM
收件人: alist-org/docs ***@***.***>
抄送: Miaoz ***@***.***>; Comment ***@***.***>
主题: Re: [alist-org/docs] zh/guide/drivers/aliyundrive (Discussion #31)


failed get link: invalid X-Device-Id，
可以访问文件夹，但是打开文件报这个错，
百度跟google都没有类似的话题，有大佬知道怎么回事嘛

―

**Reply by qqxpee** [reply]
行吧，垃圾阿里。还好天翼云盘有一模一样的备份

**Comment by haoyongbo** [comment]
failed get link: invalid X-Device-Id出错提示这个怎么处理

**Reply by iHzx** [reply]
等Alist解决吧

**Comment by yiwang-base** [comment]
阿里云盘官方已经禁止了！




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年2月13日(星期一) 晚上7:35
收件人: ***@***.***&gt;; 
抄送: "高天 ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [alist-org/docs] zh/guide/drivers/aliyundrive (Discussion #31)





 
昨天晚上弄好的阿里云盘，也发现是这样，看来不是我一个了

**Comment by yingdoutv** [comment]
{"code":500,"message":"failed get link: invalid X-Device-Id","data":null}

**Reply by dukaworks** [reply]
Same issue

**Comment by yutia3n** [comment]
failed get link: invalid X-Device-Id  原来一直可以用的，突然提示这个，用不了了。这是被阿里封号了吗。今天就是挂载了一个别人的网盘。要不就是在线削刮导致的？

**Comment by Crazyxueqiang** [comment]
failed get link: invalid X-Device-Id
有朋友解决吗，可以削刮到视频图片，文件列表可以拉取，但是播放就错误。更新Token后依旧不行。

**Comment by MR-0479** [comment]
failed get link: invalid X-Device-Id
这个问题怎么解决？

**Comment by appe233** [comment]
看来大家都出现“failed get link: invalid X-Device-Id”这个错误了，我觉得有可能是阿里云那边的问题了

**Comment by Druchii236641237** [comment]
阿里云盘用不了了,我这里显示RPC服务器不可用, 并且我验证了 alist "存储" 里面的 REFRESH TOKEN 自己会变化, 每次点击编辑都出来不一样的TOKEN, 并不是我输入的TOKEN, 非常奇怪

**Reply by Nicko8** [reply]
刷新两个字，理解一下

**Comment by 9390041** [comment]
Device id怎么拿

**Comment by liuyunfei666** [comment]
保存设置的时候弹出:"Failed init storage: The resource share_link cannot be found. Please check."

**Comment by liuyunfei666** [comment]
Failed init storage: The resource share_link cannot be found. Please check.

**Comment by liuyunfei666** [comment]
Failed init storage: The resource share_link cannot be found. Please check.

**Reply by Nicko8** [reply]
近期情况说明一下， 目前阿里云盘，读自己的文件会出现 “ invalid X-Device-Id”，这应该是阿里方面加了设备ID认证
我试的是自己内部的文件，不是阿里分享
但同时，外挂的小雅（阿里分享）却能正常使用，所以挂分享可能没问题？

**Reply by cancundeyingzi** [reply]
> 近期情况说明一下， 目前阿里云盘，读自己的文件会出现 “ invalid X-Device-Id”，这应该是阿里方面加了设备ID认证 我试的是自己内部的文件，不是阿里分享 但同时，外挂的小雅（阿里分享）却能正常使用，所以挂分享可能没问题？

别说明了。。早修复了

**Reply by wanglei52076** [reply]
怎么修复的 能说一下吗？

**Comment by wugenweixi** [comment]
failed get objs: failed to list objs: invalid X-Device-Id

**Reply by ming1101** [reply]
+1

**Reply by ZJF-Kedaya** [reply]
+1

**Reply by whumrc** [reply]
+1

**Reply by AfreyAwsome** [reply]
+1

**Reply by AfreyAwsome** [reply]
用最新版的,已经可以正常挂载阿里云盘了

**Reply by whumrc** [reply]
更新到3.11好了

**Reply by Jerryzhao26** [reply]
也没好啊。。。

**Comment by d0ndaniel** [comment]
之前都是正常的，今天突然提示“failed get objs: failed to list objs: invalid X-Device-Id”，重新获取了refresh token也不好用，应该怎么处理呀大佬们？

**Reply by d0ndaniel** [reply]
人傻了 没看到最新评论 看来大家都这样了

**Comment by bestxiaoyuxy** [comment]
failed get objs: failed to list objs: invalid X-Device-Id
怎么办？

**Comment by nathanhsuuu** [comment]
failed get objs: failed to list objs: invalid X-Device-Id

**Reply by nathanhsuuu** [reply]
已解决，更新为V3.11

**Comment by lztttt** [comment]
failed get objs: failed to list objs: invalid X-Device-Id

**Reply by nathanhsuuu** [reply]
更新V3.11，cmd--alist restart

**Comment by frankstory** [comment]
failed get objs: failed to list objs: invalid X-Device-Id

**Reply by xiyuyeyun** [reply]
failed get objs: failed to list objs: Device offline
这是啥意思？
妈的问题不断，真不能好好玩了

**Reply by nathanhsuuu** [reply]
> failed get objs: failed to list objs: Device offline 这是啥意思？ 妈的问题不断，真不能好好玩了

更新为V3.11即可，甚至可能不需要修改刷新token

**Reply by xiyuyeyun** [reply]
已经3.11了，token也换了，一直是这个啊，头大

**Reply by codeCqz** [reply]
如果出现 ：failed get link: Device offline  说明你的登录设备满了
解决办法：需要打开阿里云盘手机端 我的  --- 设置  --- 账号与安全  --- 登录设备管理
然后把所有设备删除重新配置一下就行了
如果出现一个 陌生的三星 SM-G9810 哪个是Alist登录的

**Reply by xiyuyeyun** [reply]
这个解决办法：需要打开阿里云盘手机端 我的 --- 设置 --- 账号与安全 --- 登录设备管理
然后把所有设备删除重新配置一下就行了
这个方法试了，不行！。。。。。再说和云盘手机端有什么 关系呢？

**Comment by leoz9** [comment]
failed get objs: failed to list objs: invalid X-Device-Id
这个怎么解决啊，用了新的token

**Reply by codeCqz** [reply]
更新至最新版重启软件即可

**Comment by lianqiu45** [comment]
阿里云盘挂载后打不开显示failed get objs: failed to list objs: Device offline是什么原因?

**Reply by lianqiu45** [reply]
重启一下好了,已解决.

**Comment by KimmyLueng** [comment]
failed get objs: failed to list objs: not found device info

怎样解决？

**Comment by surunze1017** [comment]
挂载阿里云盘分享链接，填写的refresh token会自动更改，每次点进去都不一样，过半天左右就显示refresh token failed了

**Reply by wdy1218** [reply]
一样

**Comment by bestxiaoyuxy** [comment]
failed get objs: failed to list objs: not found device info

**Comment by Eggplant4363** [comment]
failed get objs: failed to list objs: not found device info

**Comment by midsummer0206** [comment]
failed get objs: failed to list objs: not found device info

**Reply by xiyuyeyun** [reply]
更新到最新版 ，添加后文件还是读取不出来显示：failed get objs: failed to list objs: Device offline
怎解决

**Reply by xiyuyeyun** [reply]
重启也没用

**Reply by xiyuyeyun** [reply]
这一版本alist是真垃圾，问题 不断

**Reply by midsummer0206** [reply]
用阿里云盘open 就是不支持阿里官方的预览了

**Comment by uuid2023** [comment]
可不可以支持创建一个分享目录填写多个分享id

**Reply by uuid2023** [reply]
或者批量添加分享ID

**Comment by yimouleng** [comment]
failed get objs: failed to list objs: not found device info

**Comment by Jusman1998** [comment]
Alist Version / Alist 版本

v3.12

Driver used / 使用的存储驱动

阿里云盘分享

Describe the bug / 问题描述

目前挂载了别人的分享，通过potplayer或者电视去观看的时候，速度不到200k，这是否和阿里云open有关系，因为之前我没用阿里云open，我看我自己的盘里的内容也不到200k，换了阿里云open速度上去了，如果和阿里云open有关系，那以后的版本更新是否会解决这一问题，还是以后阿里云盘分享就只能是个摆设了？

**Reply by xiyuyeyun** [reply]
failed get objs: failed to list objs: *************8
什么原因呢？

**Reply by wolfit50** [reply]
没法用了

**Comment by yzengwen** [comment]
弱弱的问一下，这里配置阿里云盘分享是干什么？要分享的话，在阿里云盘里操作要分享的文件夹不就可以了吗？这里分享是有什么特殊用处？

**Reply by cancundeyingzi** [reply]
...挂别人999TB视频分享链接用

**Reply by Eggplant4363** [reply]
兄弟，给个链接看看

**Reply by yzengwen** [reply]
嗖嘎 你们太坏了，白嫖到这个份儿上 哈哈

**Comment by lovesnsfi** [comment]
唉，阿里云盘的分享太可惜了

**Comment by yxs520631** [comment]
invalid X-Device-Id  阿里挂在不了

**Comment by WKRZY** [comment]
不能分享挂载少了很多资源哇

**Reply by xiyuyeyun** [reply]
阿里云盘open也不好用啊，动不动就failed to refresh token: Too Many Requests
有没有其他挂载方法？

**Comment by uestchang** [comment]
用的最新的桌面版，阿里云盘上传docx文档后，在列表里不显示

**Comment by xingna97** [comment]
出现错误：failed get objs: failed to list objs: not found device info。之前一直正常，最近没用，其它网盘都没问题，但是阿里的网盘打不开了，出现上面的错误提示，不知应该怎么解决？

**Comment by Winyer** [comment]
复制报错


failed get src [/亮剑 1080i/JSTV.Drawing.Sword.E26.1080i.HDTV.MPEG2-ys333666.ts] file: failed get parent list: failed get dir: failed get parent list: failed get dir: failed get parent list: failed get dir: failed get parent list: failed to list objs: Post "https://open.aliyundrive.com/adrive/v1.0/openFile/list": net/http: TLS handshake timeout

**Comment by tcys1** [comment]
二维码刷不出来 了

**Comment by tcys1** [comment]
二维码刷不出来

**Comment by Sundy745** [comment]
Failed init storage: A403JT:Invalid JWT: other exception 有没有一样的？

**Comment by mcmoyu** [comment]
InvalidParameter.FileId:The input parameter file_id is not valid. get file request file_id should not be empty

**Comment by Askfk** [comment]
现在挂载的阿里云分享视频观看时速度只有100~300k/s，有大哥解除了这个限制吗？不然挂载了分享也基本没法看了

**Comment by lovesnsfi** [comment]
现在还有这个限制吗？发自我的手机-------- 原始邮件 --------发件人： Askfk ***@***.***>日期： 2024年1月20日周六 14:39收件人： alist-org/docs ***@***.***>抄送： 二当家的 ***@***.***>, Comment ***@***.***>主    题： Re: [alist-org/docs] zh/guide/drivers/aliyundrive (Discussion #31)
现在挂载的阿里云分享视频观看时速度只有100~300k/s，有大哥解除了这个限制吗？不然挂载了分享也基本没法看了

**Reply by Askfk** [reply]
分享的还有限制

**Comment by wisdom-wan** [comment]
哎哟，那是不是说现在都挂载不了别人的分享链接了。。。。。我自己的阿里盘都满了，有大佬知道怎么看别人分享链接里的资源吗orz

**Reply by juejijianghuaa** [reply]
可以用我的 我魔改了下（github有）

**Reply by wisdom-wan** [reply]
宝宝！！虽然我还没开始用，但是真的特别感谢你！！！

**Reply by Krssrujin** [reply]
> 可以用我的 我魔改了下（github有）
魔改的怎么用呀，可以出个教程嘛

**Reply by Krssrujin** [reply]
> 哎哟，那是不是说现在都挂载不了别人的分享链接了。。。。。我自己的阿里盘都满了，有大佬知道怎么看别人分享链接里的资源吗orz

我也是自己阿里盘满了，看不了视频了

**Reply by ZhangRuoYang666** [reply]
我用了楼上说的魔改的  能看到目录和文件  就是没办法下载和观看视频  所以也是鸡肋

**Reply by juejijianghuaa** [reply]
> 我用了楼上说的魔改的  能看到目录和文件  就是没办法下载和观看视频  所以也是鸡肋

需要同时登陆阿里open 才行，因为阿里分享用的是阿里open接口

**Comment by YWEER** [comment]
failed get objs: failed to list objs: Device offline 什么原因

**Comment by shahermark1** [comment]
二维码出不来

**Comment by ChricLu** [comment]
请愿把阿里云分享的挂载加回来呀，网上大部分资源都在阿里云夸克，夸克官方给的空间容量吝啬得要死，只能依靠阿里云了T T

**Comment by xiaoxiyao** [comment]
这个驱动受限速影响吗
